README.md

```diff
@@ -56,172 +67,74 @@ FilterHeadersStatus ExampleContext::onRequestHeaders(uint32_t headers, bool end_
 void ExampleContext::onDone() { logInfo("onDone " + std::to_string(id())); }
 ```
 
-### Compiling with the Docker build image
-
-Run docker:
+BUILD file (BUILD):
 
-```bash
-docker run -v $PWD:/work -w /work  wasmsdk:v2 /build_wasm.sh
 ```
+load("@proxy_wasm_cpp_sdk//bazel/wasm:wasm.bzl", "wasm_cc_binary")
 
-### Caching the standard libraries
-
-The first time that emscripten runs it will generate the standard libraries.  To cache these in the docker image,
-after the first successful compilation (e.g myproject.cc above), commit the image with the standard libraries:
-
-```bash
-docker commit `docker ps -l | grep wasmsdk:v2 | awk '{print $1}'` wasmsdk:v2
+wasm_cc_binary(
+    name = "my_extension.wasm",
```

Comment:
We should probably use `myplugin.wasm` here, since most of the Proxy-Wasm ecosystem standardized on plugin and not extension (since it wasn't always clear whether extension refers to extension to the ABI or the custom module).

Reply:
Done.